src/passes/pass.cpp

```diff
@@ -698,6 +698,10 @@ void PassRunner::handleAfterEffects(Pass* pass, Function* func) {
 int PassRunner::getPassDebug() {
   static const int passDebug =
     getenv("BINARYEN_PASS_DEBUG") ? atoi(getenv("BINARYEN_PASS_DEBUG")) : 0;
+  int a = 0;
+  if (true)
+    a = 5;
```

Comment:
Brace style error that triggers Travis CI error. Will not be included in the final commit